js/main.js

```diff
@@ -1,5 +1,56 @@
 'use strict';
 
 {
-  
+  const btn = document.getElementById('btn');
+
+  btn.addEventListener('click', () => {
+
+    const elemFizzNum = document.getElementById('fizz');
+    const fizzNum = elemFizzNum.value;
+    const elemBuzzNum = document.getElementById('buzz');
+    const buzzNum = elemBuzzNum.value;
+
+    const outputArea = document.getElementById('output');
+    outputArea.innerHTML = '';
+    const ptag = document.createElement('p');
+    ptag.textContent = '【出力】';
+    outputArea.appendChild(ptag);
+
+    function error() {
+      const message = document.createElement('p');
+      message.textContent = '整数値を入力してください';
+      const div = document.querySelector('div');
+      outputArea.appendChild(message);
+    }
+
+    if (isNaN(fizzNum) || isNaN(buzzNum)) {
+      return error();
+    }
+
+    if (fizzNum.match(/^-?[0-9]+\.[0-9]+$/) || buzzNum.match(/^-?[0-9]+\.[0-9]+$/)) {
+      return error();
+    }
+
+    if ((fizzNum == 0) || (buzzNum == 0)) {
```

Comment:
```suggestion
    if ((fizzNum === 0) || (buzzNum === 0)) {
```

比較には `===` を用いて厳密な比較を行いましょう。